extensions/wikia/ArticleVideo/ArticleVideoController.class.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+class ArticleVideoController extends WikiaController {
+	public function index() {
+		$wg = F::app()->wg;
+
+		$articleId = RequestContext::getMain()->getTitle()->getArticleID();
+		$thumbnailPath = $wg->videoMVPArticles[$wg->cityId][$articleId]['thumbnailPath'];
+
+		$this->setVal( 'thumbnailUrl', $wg->extensionsPath . $thumbnailPath );
+		$this->setVal( 'closeIconUrl', $wg->extensionsPath . '/wikia/ArticleVideo/images/close.svg' );
+		$this->setVal( 'videoPlayButtonUrl', $wg->extensionsPath . '/wikia/ArticleVideo/images/play-button-solid.svg' );
```

Comment:
I think it's good for now but in the future we should use `wds-icons-play` from http://fandomdesignsystem.com/#/components/assets. It doesn't have the shadow, though.